src/plugins/vis_types/timeseries/server/lib/vis_data/response_processors/series/time_shift.js

```diff
@@ -27,33 +27,16 @@ export function timeShift(
         const offsetValue = matches[1];
         const offsetUnit = matches[2];
 
-        const defaultTimezone = moment().zoneName();
```

Comment:
Let's split this out please - it's not related to the rest of the PR and I would like to have a separate PR to point to for this (and also test it better without blocking this PR).